flink-table/flink-table-common/src/main/java/org/apache/flink/table/types/inference/strategies/SpecificInputTypeStrategies.java

```diff
@@ -65,6 +65,11 @@ public static InputTypeStrategy windowTimeIndicator(TimestampKind timestampKind)
         return new WindowTimeIndictorInputTypeStrategy(timestampKind);
     }
 
+    /** See {@link WindowTimeIndictorInputTypeStrategy}. */
+    public static InputTypeStrategy windowTimeIndicator() {
+        return new WindowTimeIndictorInputTypeStrategy(null);
```

Comment:
Is there a good example of when this is null?

Is this already tested?

Reply:
When we pass it as null? The parameter is used to configure the behaviour of the `WindowTimeIndictorInputTypeStrategy` class. If we configure it with `null` we don't perform the specific checks for `PROCTIME`, `ROWTIME` attributes. We just check that the argument is of any of the two.